pom.xml

```diff
@@ -25,7 +25,7 @@
 	<properties>
 		<openMRSVersion>2.3.1</openMRSVersion>
 		<htmlformentryVersion>3.11.0</htmlformentryVersion>
-		<addresshierarchyrwandaVersion>1.2.9-SNAPSHOT</addresshierarchyrwandaVersion>
+		<addresshierarchyVersion>2.12.0</addresshierarchyVersion>
```

Comment:
Is there any reason why you did not use 2.13.0?

Reply:
@dkayiwa I didn't notice there was a 2.13.0 release! It's [fixed now.](https://github.com/Rwanda-EMR/openmrs-module-rwandaprimarycare/pull/9/commits/d8ab9ad50ab34496e9b027557e9e7c0ea5c731ce)